lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -169,13 +150,15 @@ function dosomething_rogue_get_by_file_id($fid)
  * phoenix and it's corresponding id's in Rogue
  *
  * @param string $rbid
- * @param string $fid
  * @param object $rogue_reportback
  *
  * @return InsertQuery object
  */
-function dosomething_rogue_store_rogue_references($rbid, $fid, $rogue_reportback)
+function dosomething_rogue_store_rogue_references($rbid, $rogue_reportback)
 {
+  $reportback = entity_load_unchanged('reportback', [$rbid]);
+  $fid = array_pop($reportback->fids);
```

Comment:
@chloealee this helped me notice an error in my PR oh boy! but I thiiiiink that would be pretty helpful? not if it would be a huge pain though